src/App.tsx

```diff
@@ -1,10 +1,55 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { Header } from './Components/Header';
+import { TodoList } from './Components/TodoList';
+import { Footer } from './Components/Footer';
+import { Notifications } from './Components/Notifications';
+import { FilterType } from './types/FilterType';
+
+const USER_ID = 6677;
+
+const handleFilteringTodos = (todos: Todo[], filterType: FilterType) => {
+  const filteredTodos = todos.filter(todo => {
+    switch (filterType) {
+      case FilterType.ACTIVE:
+        return !todo.completed;
+      case FilterType.COMPLETED:
+        return todo.completed;
+      case FilterType.ALL:
+        return true;
+      default:
+        return todo;
+    }
+  });
+
+  return filteredTodos;
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.ALL);
+  const [error, setError] = useState(false);
```

Comment:
```suggestion
  const [isError, setIsError] = useState(false);
```
![image](https://user-images.githubusercontent.com/96409563/226275321-6b900a74-602c-4c72-b659-b02fc3e4547f.png)